tests/framework/web/session/SessionTest.php

```diff
@@ -88,4 +88,20 @@ public function testSetName()
 
         $session->destroy();
     }
+
+    /**
+     * This test ensures that regnerate ID will actually generate a new ID.
+     */
+    public function testRegenerateId()
+    {
+        $session = new Session();
+
+        $before = $session->getId();
+
+        $session->regenerateID();
+
+        $after = $session->getId();
+
+        $this->assertNotEquals($before, $after);
```

Comment:
That's not what is documented in `regenerateID()` phpdoc.

Reply:
@samdark Note that documentation between https://github.com/yiisoft/yii2/blob/master/framework/web/Session.php#L287 and child class https://github.com/yiisoft/yii2/blob/master/framework/web/DbSession.php#L99 is not the same. (should it use inheritDoc?)

Reply:
Yes, it should.